dev/io.openliberty.http.monitor/src/io/openliberty/http/monitor/metrics/MetricsManager.java

```diff
@@ -69,7 +69,7 @@ public static MetricsManager getInstance() {
      * @param appName The appName, can be null to indicate server context
      */
 	public void updateHttpMetrics(HttpStatAttributes httpStatAttributes, Duration duration, String appName) {
```

Comment:
The `appName` in the parameter is not used.

Reply:
Fixed.